lib/time/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TIME
+	bool "System real-time library"
+
+if TIME
+
+config TIME_MIN_YEAR
+	int "Minimum year to support for real-time services"
+	default 2000
```

Comment:
Not really understanding why the defaults are so limited here.  The rtc_time struct is basically just a Unix struct tm and the year field is a 32 bit signed int with the zero at 1900.